app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -766,11 +766,15 @@ def build_toolbar_hide_button(id)
     when "Service"
       return build_toolbar_hide_button_service(id)
     when "Vm"
+      # maybe use .active? instead of .orphaned?
+      # vm_or_template.rb #1331
       case id
       when "vm_clone"
         return true unless @record.cloneable?
+        return true if @record.orphaned?
```

Comment:
@chessbyte final changes for this BZ will be different. Buttons should be disabled with explaining hover text, while this change is hiding them completly